src/comp_loinc/groups/groups_builders.py

```diff
@@ -243,11 +248,11 @@ def do_abstracts2(self):
         module: Module = self.runtime.current_module
         for c in _more_than_components:
             loinc_term: LoincTerm = module.getsert_entity(
-                entity_id=f"comploinc:/group/component/{c.part_number}",
+                entity_id=f"{ComploincNodeType.group_node.value.id_prefix}:group/component/{c.part_number}",
```

Comment:
Do you think this will result in a change in the ultimate URIs that show up in `.owl`?

If so, we/I should update the the prefix maps in `PREFIXES_METRICS` in the `makefile`.

Reply:
The identifiers in `networkx` are now CURIE based. this ID is also used in LinkML.